src/Lecture1.hs

```diff
@@ -20,6 +20,7 @@ It's absolutely okay if you feel that your implementations are not
 perfect. You can return to these exercises after future lectures and
 improve your solutions if you see any possible improvements.
 -}
+{-# OPTIONS_GHC -Wno-incomplete-patterns #-}
```

Comment:
Usually, it's better not to disable this warning but fix all such warnings instead 🙂